controller/logs.go

```diff
@@ -27,6 +29,7 @@ func (c *Controller) GetActiveDeploymentLogs(ctx context.Context, numLines int32
 	latestDeploy := deployments[0]
 	// Streaming
 	prevIdx := 0
+	status := "SUCCESS"
```

Comment:
Should this come from the `latestDeploy`?

Reply:
Oh tru